src/RefreshIndicator/RefreshIndicator.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Paper from '../Paper';
+import { CircularProgress } from '../Progress';
+import withStyles from '../styles/withStyles';
+
+const SIZE = 50; // same as CircularProgress
+
+export const styles = theme => ({
+  root: {
+    borderRadius: '50%'
+  }
+})
+
+class RefreshIndicator extends React.Component {
+  getPaddingSize() {
+    const padding = this.props.size * 0.14;
```

Comment:
Shouldn't we return padding directily.

return this.props.size * 0.14

The syntax used in this function and getPaperSize should be same